var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -12,13 +11,17 @@ class Arbor(CMakePackage, CudaPackage):
 
     homepage = "https://arbor-sim.org"
     git = "https://github.com/arbor-sim/arbor.git"
-    url = "https://github.com/arbor-sim/arbor/releases/download/v0.8.1/arbor-v0.8.1-full.tar.gz"
-
-    maintainers("thorstenhater", "brenthuisman")
+    url = "https://github.com/arbor-sim/arbor/releases/download/v0.8.1/arbor-v0.10.0-full.tar.gz"
+    maintainers = ("thorstenhater", "boeschf")
     license("BSD-3-Clause")
 
     version("master", branch="master")
     version("develop")
```

Comment:
There is no `develop` branch AFAICT.

Reply:
No, but it is required in case you build against an arbitrary checkout not corresponding to an upstream branch (i.e. testing of a PR). This is (or was) a designated workaround IIRC.

Reply:
I hadn't heard about that workaround.

Reply:
`dev-build` requires a version name, and the only way to force it not to lookup a git branch with that name is (or was) this: an otherwise empty version declaration. See usage here: https://github.com/arbor-sim/arbor/commit/7d36bebc41158968aacb62b82e9fb67cd73d8ba0#diff-1a4dd20939aab338a06244dd0139c9249b60d1d5994317e4e9f256437cc37b03R65

If there is a newer and better way to do this, do tell :) Maybe `local_source` would be a more descriptive name actually.